test/e2e/utilities.js

```diff
@@ -213,7 +215,6 @@ const afterTestClear = (params) => {
     process.kill(btcdPid);
     process.kill(fundsLndPid);
     rimraf.sync(params.userPath);
```

Comment:
I forgot to add check existance of folder, so maybe need to wrap `rimraf` in something like 
```
    if (fs.existsSync(params.userPath)) {
        rimraf.sync(params.userPath);
    }
```